virga/root_functions.py

```diff
@@ -171,6 +171,119 @@ def vfall(r, grav,mw_atmos,mfp,visc,
 
     return vfall_r 
 
+
+ def vfall_aggregrates(r, grav, mw_atmos, t, p, rhop, D=2, Ragg=r):
```

Comment:
Better way to make a default is `Ragg=None` then within the function: 

`if isinstance(Ragg, type(None)): Ragg=r`